views/pages/admin/generic-editor.pug

```diff
@@ -31,6 +31,10 @@ block body
     div(id="body")
       h2(id="editor-title")=editorName
       div(id="jsoneditor" data-source=editorSource)
+      div(class="open-editor-container")
+        button(class="pink-button disabled open-editor-button" id="open-editor-btn" disabled="true")
```

Comment:
Since we're using the `pink-button`s, can we use the `+pink-button()` mixin instead?

Reply:
Pink button doesn't support extra classes or attributes like "disabled." I could just set the extra stuff in the ts file, but then we have another case of basically created a button in ts

Reply:
sounds good!  I might rework the pink button later to be able to handle that functionality, but this works for now

Reply:
thanks! i wanted to try n tackle it, but didn't want to break the usage of the pink button in multiple places